src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
```

Comment:
It's a good practice to remove commented code or unused code. Please remove the line `<>Put the list here</>` since it's not needed anymore 🧹

Reply:
![image](https://user-images.githubusercontent.com/57044635/233790297-fcec0f33-2074-4b34-ab41-87ea802c094a.png)

She is not there anyway.